src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +16,37 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    private static final String SEARCH_LETTER = "w";
+    String[] emptyArray = new String[0];
+    List<String> wordsList = new ArrayList<String>();
+
     public String[] readFromFile(String fileName) {
-        return null;
+        String[] words = new String[0];
+        try (BufferedReader br = new BufferedReader(new FileReader(fileName))) {
+            StringBuilder result = new StringBuilder();
+            String line = br.readLine();
+            while (line != null) {
+                result.append(line);
+                line = br.readLine();
+            }
+            words = result.toString().split(" ");
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("Can't read file", e);
+        } catch (IOException exception) {
+            exception.printStackTrace();
+        }
+
+        for (String word : words) {
+            word = word.toLowerCase().replaceAll("[^A-Za-z]", "");
+            if (word.indexOf(SEARCH_LETTER) == 0) {
```

Comment:
let's make this manipulation while reading files. This way you will get rid of using ArrayList